modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -156,7 +156,11 @@ private fun WhatsNewPageLoaded(
                 modifier = Modifier.padding(horizontal = 32.dp),
             )
 
-            Spacer(modifier = Modifier.height(32.dp))
+            if (state.fullModel) {
+                Spacer(modifier = Modifier.weight(0.5f))
+            } else {
+                Spacer(modifier = Modifier.height(32.dp))
+            }
```

Comment:
I added this code back in order to put the tittle in center as we were doing before sharing clip whats new

Reply:
This seems like it could be more centred. Would it be possible to use vertical alignment?

| Before | After |
| --- | --- |
| ![Screenshot_20241128_162120](https://github.com/user-attachments/assets/18ea3636-2091-4cb5-abfd-1030353a8a1f) | ![shuffle-after](https://github.com/user-attachments/assets/bf48bdfb-7c0b-4949-9d8f-fd7f27909f95) | 

```kotlin
            if (state.fullModel) {
                Spacer(Modifier.height(8.dp))

                Pill()

                Box(
                    modifier = Modifier.align(Alignment.Start),
                ) {
                    RowTextButton(
                        text = stringResource(LR.string.cancel),
                        fontSize = 15.sp,
                        onClick = performClose,
                        fullWidth = false,
                        includePadding = false,
                    )
                }
            }

            Column(
                horizontalAlignment = Alignment.CenterHorizontally,
                verticalArrangement = Arrangement.Center,
                modifier = Modifier.fillMaxWidth().weight(1f) ,
            ) {
                header()

                Spacer(modifier = Modifier.height(16.dp))

                TextH10(
                    text = stringResource(id = state.feature.title),
                    textAlign = TextAlign.Center,
                    color = MaterialTheme.theme.colors.primaryText01,
                    modifier = Modifier.padding(horizontal = 32.dp),
                )

                Spacer(modifier = Modifier.height(8.dp))

                TextP40(
                    text = stringResource(state.feature.message),
                    textAlign = TextAlign.Center,
                    color = MaterialTheme.theme.colors.primaryText01,
                    modifier = Modifier.padding(horizontal = 32.dp),
                )
            }

            RowButton(
                text = stringResource(state.feature.confirmButtonTitle),
                onClick = onConfirm,
                includePadding = false,
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(horizontal = 16.dp),
            )
```

Reply:
That looks better! I also added a small improvement to your suggestion for small device in landscape